.github/workflows/actions.yml

```diff
@@ -9,167 +9,177 @@ jobs:
   # main branch and pull request branch
   changes:
     runs-on: ubuntu-latest
-    outputs:
-      core: ${{ steps.filter.outputs.core }}
-      client: ${{ steps.filter.outputs.client }}
-      transport: ${{ steps.filter.outputs.transport }}
-      server: ${{ steps.filter.outputs.server }}
-      commit-author: ${{ steps.check.outputs.author }}
+    # outputs:
```

Comment:
you need to remove the stuff that is currently commented out.

Reply:
Yeah that was just for cat-ing .git/config, I'll revert it back. I'm still working on getting the tokens to work, and if that doesn't work then I'll try the GitHub app approach. Until then I'll mark this PR as a draft.